registry/handlers/app_test.go

```diff
@@ -185,23 +186,25 @@ func TestNewApp(t *testing.T) {
 		t.Fatalf("unexpected status code during request: %v", err)
 	}
 
-	if req.Header.Get("Content-Type") != "application/json; charset=utf-8" {
-		t.Fatalf("unexpected content-type: %v != %v", req.Header.Get("Content-Type"), "application/json; charset=utf-8")
-	}
+	/*
+		if req.Header.Get("Content-Type") != "application/json; charset=utf-8" {
```

Comment:
not sure why I removed it, but whatever the reason it works now so I added it back in.